### PR TITLE
Modified fizzbuzz.livemd

### PR DESCRIPTION
**Pull request by aryanlath**
I have added the Tom Scott Fizz Buzz video to the FIzz Buzz Exercise.

### CODE REVIEW

File: exercises/fizzbuzz (2).livemd [hunk 1]
```diff
@@ -5,7 +5,7 @@ Mix.install([
   {:youtube, github: "brooklinjazz/youtube"},
   {:hidden_cell, github: "brooklinjazz/hidden_cell"},
   {:tested_cell, github: "brooklinjazz/tested_cell"},
-  {:utils, path: "#{__DIR__}/../utils"}
+<!--   {:utils, path: "#{__DIR__}/../utils"} -->
```
Comment (BrooklinJazz) [inline exercises/fizzbuzz (2).livemd:8]: Can you uncomment this line?

**Review by BrooklinJazz — CHANGES_REQUESTED** [review]
There are two .DS_Store files included in this PR, as well as a commented setup line. Can you remove them? After that, this should be good to merge.

File: exercises/fizzbuzz (2).livemd (hunk 1 above)
Comment (aryanlath) [reply exercises/fizzbuzz (2).livemd:8]: Sure

**Comment by aryanlath** [comment]
I have made all the changes you mentioned

**Comment by BrooklinJazz** [comment]
Thank you!

**Pull request merged**